### ISSUE TITLE
Default price per contact

### ISSUE THREAD

**Issue by marcobischoff** [hql287/Manta#213]
Please answer these question so I can help you faster.

**Is it a bug or a feature request?**

- [ ] It's a bug.
- [  x ] I'm requesting a new feature.

**Which version are you using?**

- [  x ] macOS
- [ ] Windows
- [ ] Linux

**Which is the version/distribution of the OS?**
macOS High Sierra

**Describe your issue**
Is it possible to have a default price per contact, as it could be that you have rates which applies to all items.

**Comment by hql287** [comment]
@marcobischoff I thought about this. I think this is called group pricing (If I understand you correctly). So if an item is $10 and the contact belongs to a group called VIP-1 (which will automatically apply a 10% discount to all products/services) he/she will only need to pay $9 for that item. 

If this is what you want, I think we'll need to wait until the Product/Service feature is implemented. You can follow the progress here: https://github.com/hql287/Manta/issues/111

@rwachtler: Any update on this? :)

**Comment by rwachtler** [comment]
@hql287 Yup, I implemented already an additional component for the services similar to your `<ItemList />` component and some of the actions.
What already works:
- The "Services" tab is already there
- Adding and removing and rearranging element already works (it's also within the store)

What I still have to do:
- Make the changes persistent within the database
- Add a select input field to the "New contract" view (Should I completely remove the form for the Services/Products here? So the user will be only able to select from existing service entries, or should we leave the form there and the select list below?)
-- In the first case, where only a select dropdown is there, we'd probably need some additional hint to guide the user to the services tab.
- Write tests
- Maybe also some minor optimizations & cleanup

You can also check out the latest changes I did on my fork, the latest stuff should be pushed. Sorry for delaying this feature.

**Comment by marcobischoff** [comment]
@hql287 it thought more of something, if you do several tasks for the same price for the same customer.
Make this sense to you?

**Comment by hql287** [comment]
Hey guys, sorry for taking a bit long to get back to you. We're celebrating Lunar New Year here in S.E.A. Pretty much like Thanksgiving and Christmas, lots of food means slow in response 😂

Anyway, @rwachtler Will check out your fork today. Can you allow me to push to your fork? Thanks!

> do several tasks for the same price for the same customer.
@marcobischoff : Can you give a more detailed example? I'm not sure I understand what you mean.

**Comment by marcobischoff** [comment]
Lets try it: A user has to write reports to the client as he sell his work on hourly rates. So he would fill out the description the time he used for this task and also the price. It would be a lot more comfortable if he only needs to fill out the description and the hours, the price is automatically there. 

I hope it is a little bit clearer now.

**Comment by rwachtler** [comment]
@hql287 Done, I did my changes in the `feature/services` branch.
